lecture_3/03_project/controller/internal/engine/engine.go

```diff
@@ -57,8 +57,7 @@ func (e *Engine) processBetsCalculated(ctx context.Context) error {
 		return err
 	}
 
-	resultingBets := e.handler.HandleBetsCalculated(ctx, consumedBetsCalculated)
-	e.publisher.PublishBets(ctx, resultingBets)
+	e.handler.HandleBetsCalculated(ctx, consumedBetsCalculated)
```

Comment:
We do not want to change the controller producing mechanisms :)
You need to handle these messages in your code.

e.g. if you have a bet in your db skip the message :)